### DISCUSSION TITLE
set printer to heat up hotend on the side of hotbed

### DISCUSSION THREAD

**Discussion by dannler** [Guilouz/Klipper-Flsun-Speeder-Pad#106] [Q&A]
hi there, where can i change the firmware to make the printer to heat up on the side of the printbed instead of the center, i would like to avoid the spillage in the center of the printbed, and i would like to do such a setting on the printer itself and not the slicer.

i have gone trough the config files and didnt see anything there to adjust

**Comment by dev-zetta** [comment]
You need to set your center, i.e. Homing location.

**Reply by dannler** [reply]
if i set my center/  home location to the side, and not in the center, wont it cause issues with calibrations ect?

**Comment by dannler** [comment]
i was supplied with the proper g-code thanks
